listenbrainz/db/user.py

```diff
@@ -492,10 +492,12 @@ def get_similar_users(user_id: int) -> SimilarUsers:
             'user_id': user_id,
         })
         row = result.fetchone()
-        users = {}
-        for user in row[1]:
-            users[user] = row[1][user][0]
-        return SimilarUsers(user_id=row[0], similar_users=users) if row else None
+        if row:
+            users = {}
+            for user in row[1]:
```

Comment:
not directly related to this change, but can we use dict access here to make it clearer what we're selecting? `row['similar_users']`